lib/widgets/code_block.dart

```diff
@@ -1,267 +1,549 @@
+      vi: TextStyle(color: const HSLColor.fromAHSL(1, 241, 0.68, 0.28).toColor()),
+
+      // .il { color: hsl(0deg 0% 40%); }
+      il: TextStyle(color: const HSLColor.fromAHSL(1, 0, 0, 0.40).toColor())
+    );
+  }
+
+  CodeBlockTextStyles._({
+    required this.plain,
+    required TextStyle hll,
+    required TextStyle c,
+    required TextStyle err,
+    required TextStyle k,
+    required TextStyle o,
+    required TextStyle cm,
+    required TextStyle cp,
+    required TextStyle c1,
+    required TextStyle cs,
+    required TextStyle gd,
+    required TextStyle ge,
+    required TextStyle gr,
+    required TextStyle gh,
+    required TextStyle gi,
+    required TextStyle go,
+    required TextStyle gp,
+    required TextStyle gs,
+    required TextStyle gu,
+    required TextStyle gt,
+    required TextStyle kc,
+    required TextStyle kd,
+    required TextStyle kn,
+    required TextStyle kp,
+    required TextStyle kr,
+    required TextStyle kt,
+    required TextStyle m,
+    required TextStyle s,
+    required TextStyle na,
+    required TextStyle nb,
+    required TextStyle nc,
+    required TextStyle no,
+    required TextStyle nd,
+    required TextStyle ni,
+    required TextStyle ne,
+    required TextStyle nf,
+    required TextStyle nl,
+    required TextStyle nn,
+    required TextStyle nt,
+    required TextStyle nv,
+    required TextStyle nx,
+    required TextStyle ow,
+    required TextStyle w,
+    required TextStyle mf,
+    required TextStyle mh,
+    required TextStyle mi,
+    required TextStyle mo,
+    required TextStyle sb,
+    required TextStyle sc,
+    required TextStyle sd,
+    required TextStyle s2,
+    required TextStyle se,
+    required TextStyle sh,
+    required TextStyle si,
+    required TextStyle sx,
+    required TextStyle sr,
+    required TextStyle s1,
+    required TextStyle ss,
+    required TextStyle bp,
+    required TextStyle vc,
+    required TextStyle vg,
+    required TextStyle vi,
+    required TextStyle il,
+  }) :
+    _hll = hll,
+    _c = c,
+    _err = err,
+    _k = k,
+    _o = o,
+    _cm = cm,
+    _cp = cp,
+    _c1 = c1,
+    _cs = cs,
+    _gd = gd,
+    _ge = ge,
+    _gr = gr,
+    _gh = gh,
+    _gi = gi,
+    _go = go,
+    _gp = gp,
+    _gs = gs,
+    _gu = gu,
+    _gt = gt,
+    _kc = kc,
+    _kd = kd,
+    _kn = kn,
+    _kp = kp,
+    _kr = kr,
+    _kt = kt,
+    _m = m,
+    _s = s,
+    _na = na,
+    _nb = nb,
+    _nc = nc,
+    _no = no,
+    _nd = nd,
+    _ni = ni,
+    _ne = ne,
+    _nf = nf,
+    _nl = nl,
+    _nn = nn,
+    _nt = nt,
+    _nv = nv,
+    _nx = nx,
+    _ow = ow,
+    _w = w,
+    _mf = mf,
+    _mh = mh,
+    _mi = mi,
+    _mo = mo,
+    _sb = sb,
+    _sc = sc,
+    _sd = sd,
+    _s2 = s2,
+    _se = se,
+    _sh = sh,
+    _si = si,
+    _sx = sx,
+    _sr = sr,
+    _s1 = s1,
+    _ss = ss,
+    _bp = bp,
+    _vc = vc,
+    _vg = vg,
+    _vi = vi,
+    _il = il;
+
+  final TextStyle plain;
+
+  final TextStyle _hll;
```

Comment:
`plain` here has a different role from all the other styles, in that it's meant to be a baseline that each of them gets applied on top of.

So that asymmetry would be good to make explicit, perhaps in dartdoc on `plain`. In particular that helps explain why in the last commit, an explicit weight gets added to `plain` but there's no need for it on the rest.